pages/embedded-viewer/[manifestId].tsx

```diff
@@ -14,9 +16,49 @@ const EmbeddedViewerPage: NextPage = () => {
   const [work, setWork] = React.useState<Work | null>();
 
   const { isWorkRestricted } = useWorkAuth(work);
-
   const thumbnail = work?.thumbnail || "";
 
+  const searchParams = getUrlSearchParams(decodeURIComponent(router.asPath));
+  console.log("router.asPath", router.asPath);
+  console.log("searchParams", searchParams);
+
+  // Filter out any object properties which contain '/embedded-viewer' in the value
+  const filteredSearchParams = Object.keys(searchParams)
+    .filter((key) => !key.includes("/embedded-viewer") && !(key === "as"))
+    .reduce(
+      (obj, key) => {
+        obj[key] = searchParams[key];
+        return obj;
+      },
+      {} as { [key: string]: string }
+    );
+  console.log("filteredSearchParams", filteredSearchParams);
+
+  const {
+    informationPanel = "true",
+    informationPanelOpen = "false",
+    informationPanelAbout = "true",
+    showTitle = "true",
+    showIIIFBadge = "true",
+  } = filteredSearchParams;
+  console.log("informationPanel", informationPanel);
+
+  const viewerOptions: ViewerConfigOptions = {
+    informationPanel: {
+      open:
+        informationPanel === "true" && informationPanelOpen === "true"
+          ? true
+          : false,
+      renderAbout:
+        informationPanel === "true" && informationPanelAbout === "true"
+          ? true
+          : false,
+      renderToggle: informationPanel === "true" ? true : false,
+    },
+    showIIIFBadge: showIIIFBadge === "true" ? true : false,
```

Comment:
What if we just set this to false since this isn't a customizable option. Might be a little less confusing to end users.
```suggestion
    showIIIFBadge: false,
```